src/irmin-pack-utils/ppcf.ml

```diff
@@ -0,0 +1,30 @@
+module Io = Irmin_pack_unix.Io.Unix
+module Io_errors = Irmin_pack_unix.Io_errors.Make (Io)
+module Control = Irmin_pack_unix.Control_file.Upper (Io)
```

Comment:
Also need a way to read control files of volumes. Maybe just an optional argument to the command line that defaults to upper.

Reply:
I had forgotten that the `Upper` was enforced here, considering the necessary functions for the `Volume` are already done, it should not be an issue to update that.

However it makes me wonder if there is a way to detect if a control file is from an `Upper` or `Volume` store, it would avoid having a redondant command line argument

Reply:
It's probably easier to have a command line argument, but you could use the `Layout.classify_filename` function to guess if it's an upper or lower volume control file (but the lower files are not currently classified?)

Reply:
If we add it to the classification (which we should! but probably a separate PR), we can use that indeed since they have different file names. :+1:

Reply:
I agree, I'll update the tool once we have the classification